<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientReserveView</class>
 <widget class="QWidget" name="PatientReserveView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预约界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* 设置背景颜色和字体样式 */
    QWidget {
        background-color: #f5f5f5;
        font-family: Arial, sans-serif;
        font-size: 12px;
    }

    /* 设置 QLabel 的样式 */
    QLabel {
        color: #333333;
        font-weight: bold;
    }

    /* 设置 QLineEdit 的样式 */
    QLineEdit {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 4px;
        background: #ffffff;
    }

    /* 设置 QComboBox 的样式 */
    QComboBox {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 4px;
        background: #ffffff;
    }

    /* 设置 QSpinBox 的样式 */
    QSpinBox {
        border: 1px solid #cccccc;
        border-radius: 4px;
        padding: 4px;
        background: #ffffff;
    }

    /* 设置 QPushButton 的样式 */
    QPushButton {
        background-color: #4CAF50;
        color: white;
        border: none;
        border-radius: 4px;
        padding: 8px 16px;
        font-size: 14px;
    }

    QPushButton:hover {
        background-color: #45a049;
    }

    QPushButton#button_cancel {
        background-color: #f44336;
    }

    QPushButton#button_cancel:hover {
        background-color: #d32f2f;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="dbEditName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_email">
       <property name="text">
        <string>邮箱</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dbEditEmail"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_doctor">
       <property name="text">
        <string>医生</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="dbComboDoctor"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>日期</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dbEditDate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>具体时间</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTimeEdit" name="dbEditTime">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btReserve">
     <property name="text">
      <string>预订</string>
     </property>
     <property name="icon">
      <iconset resource="lab4.qrc">
       <normaloff>:/img/reserve.png</normaloff>:/img/reserve.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lab4.qrc"/>
 </resources>
 <connections/>
</ui>
